homework.js

```diff
@@ -1,22 +1,46 @@
 // 1. Set the variable `givenName` to the string "Addison".
+let givenName = 'Addison';
+console.log('Addison');
 
 // 2. Set candies equal to 20, people to 6, and leftover equal
 // to the remainder of dividing 20 by 6.
+let candies = 20;
+let people = 6;
+let leftover = 20 % 6;
+console.log('leftover');
+
 
 // 3. Create a function called greeting that returns "Hello, <name>!",
 // where <name> is the name given as an argument to the function.
+function greeting(name) {
+    return "Hello, " + name  + "!"
+}
 
 // 4. Create a function called isOdd that, given a number, will
 // return true if it is odd and false if it is not. An odd number is a
 // number which, when divided by 2, has a remainder of 1 or -1.
+function isOdd(num) {
+   if (num % 2 === 1) || (num % 2 === -1);
+   else false
+   console.log('num')
+}
```

Comment:
I think the syntax you were going for here is:
```js
function isOdd (num) {
  if (num % 2 === -1 || num % 2 === 1) {
    return true
  } else {
    return false
  }
}
```

That would work, but you could also do:

```js
function isOdd (num) {
  return num % 2 === -1 || num % 2 === 1
}
```
☝️ This works because the expressions `num % 2 === -1` and `num % 2 === 1` will return a boolean value when they are evaluated, so your function will just return those boolean values.